smrf/tests/smrf_test_case.py

```diff
@@ -15,17 +17,70 @@ class SMRFTestCase(unittest.TestCase):
     The base test case for SMRF that will load in the configuration file
     and store as the base config. Also will remove the output
     directory upon tear down.
+
+    Runs the short simulation over reynolds mountain east
     """
-    dist_variables = [
+    DIST_VARIABLES = frozenset([
         'air_temp',
         'cloud_factor',
         'precip',
         'thermal',
         'vapor_pressure',
         'wind',
-    ]
+    ])
+
+    BASE_INI_FILE_NAME = 'test_base_config.ini'
+
+    test_dir = PurePath(smrf.__file__).parent.joinpath('tests')
+    config_file = os.path.join(test_dir, BASE_INI_FILE_NAME)
+
+    @property
+    def dist_variables(self):
+        if self._dist_variables is None:
+            self._dist_variables = list(self.DIST_VARIABLES)
+        return self._dist_variables
+
+    @property
+    def base_config(self):
+        return deepcopy(self._base_config)
+
+    @classmethod
+    def base_config_copy(cls):
+        return deepcopy(cls._base_config)
+
+    @classmethod
+    def load_base_config(cls):
+        cls._base_config = get_user_config(cls.config_file, modules='smrf')
+
+    @classmethod
+    def setUpClass(cls):
+        cls.load_base_config()
+        cls.create_output_dir()
+
+    @classmethod
+    def tearDownClass(cls):
+        cls.remove_output_dir()
+        delattr(cls, '_output_dir')
+
+    @classmethod
+    def create_output_dir(cls):
+        folder = os.path.join(cls._base_config.cfg['output']['out_location'])
+
+        # Remove any potential files to ensure fresh run
+        if os.path.isdir(folder):
+            shutil.rmtree(folder)
+
+        os.makedirs(folder)
+        cls._output_dir = folder
 
-    def can_i_run_smrf(self, config):
+    @classmethod
+    def remove_output_dir(cls):
+        if hasattr(cls, '_output_dir') and \
+                os.path.exists(cls._output_dir):
+            shutil.rmtree(cls._output_dir)
+
+    @staticmethod
+    def can_i_run_smrf(config):
```

Comment:
At some point I would like to see this no longer in use.

Reply:
Agreed and also remove it from `framework`

Reply:
https://github.com/USDA-ARS-NWRC/smrf/issues/182